apis/python/operator/src/lib.rs

```diff
@@ -41,7 +41,7 @@ impl<T> DelayedCleanup<T> {
         CleanupHandle(self.0.clone())
     }
 
-    pub fn get_mut(&mut self) -> std::sync::MutexGuard<T> {
+    pub fn get_mut(&self) -> std::sync::MutexGuard<T> {
```

Comment:
This will still error when used concurrently. So the error just happens in a different place now, but concurrent operations are still not possible.